app/pages/about/contact.spec.js

```diff
@@ -0,0 +1,17 @@
+/* eslint prefer-arrow-callback: 0, func-names: 0, 'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }] */
+
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import Contact from './contact';
+
+describe('Contact', function () {
+  it('renders without crashing', function () {
```

Comment:
@jelliotartz this test is empty.

Reply:
Perhaps something like `shallow(<Contact />);` would do.

Reply:
This is an empty test per suggestion from Roger in [this PR](https://github.com/zooniverse/Panoptes-Front-End/pull/4190). Look down towards the bottom of the PR, right before the commit that changes the test to an empty function, for a hidden comment that you need to open up with the show/hide outdated tab.

Reply:
In the case of AboutHome there was a before block that shallow rendered the component though, so I think Simone's suggestion is valid.

Reply:
oops, duh, thank you! you're totally right!